api/HTMLDialogElement.json

```
@@ -14,28 +14,38 @@
           "edge": {
             "version_added": "79"
           },
-          "firefox": {
-            "version_added": "53",
-            "flags": [
-              {
-                "type": "preference",
-                "name": "dom.dialog_element.enabled",
-                "value_to_set": "true"
-              }
-            ],
-            "notes": "See <a href='https://bugzil.la/840640'>bug 840640</a>."
-          },
-          "firefox_android": {
-            "version_added": "53",
-            "flags": [
-              {
-                "type": "preference",
-                "name": "dom.dialog_element.enabled",
-                "value_to_set": "true"
-              }
-            ],
-            "notes": "See <a href='https://bugzil.la/840640'>bug 840640</a>."
-          },
+          "firefox": [
+            {
+              "version_added": "98"
+            },
+            {
+              "version_added": "53",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.dialog_element.enabled",
+                  "value_to_set": "true"
+                }
+              ],
+              "notes": "See <a href='https://bugzil.la/840640'>bug 840640</a>."
+            }
+          ],
+          "firefox_android": [
+            {
+              "version_added": "98"
+            },
+            {
+              "version_added": "53",
```

Firefox for Android doesn't support preferences in stable since 79. We should close out this support statement

```suggestion
              "version_added": "53",
              "version_removed": "79",
```